control/config_proxy.go

```diff
@@ -0,0 +1,104 @@
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2015 Intel Corporation
```

Comment:
We should probably be a bit more diligent on setting the year (not like it actually matters from what I have read about having an actual copyright line in your source code) to the year the code landed, so `2016` for anything new.